crates/dreamchecker/src/lib.rs

```diff
@@ -354,11 +354,13 @@ fn run_inner(context: &Context, objtree: &ObjectTree, cli: bool) {
     objtree.root().recurse(&mut |ty| {
         for proc in ty.iter_self_procs() {
             analyzer.check_kwargs(proc);
+            analyzer.check_arg_type(proc);//count(proc);
```

Comment:
```suggestion
            analyzer.check_arg_type(proc);
```
;)

Reply:
whoops. yeah I can probably remove the check_arg_count method entirely since I replaced it with check_arg_type, and I can rename it to check_args